### ISSUE TITLE
Add validation for AWS CLI version

### ISSUE THREAD

**Issue by otterley** [aws-containers/amazon-ecs-exec-checker#25]
False alarm, closing

**Comment by otterley** [comment]
This check is indirect and doesn't _really_ check whether the CLI version is new enough:

```sh
# Check whether the AWS CLI v1.19.28/v2.1.30 or later exists
executeCommandEnabled=$(echo "${describedTaskJson}" | jq -r ".tasks[0].enableExecuteCommand")
if [[ "x${executeCommandEnabled}" = "xnull" ]]; then
  printf "${COLOR_RED}Pre-flight check failed: ECS Exec requires the AWS CLI v1.19.28/v2.1.30 or later.\n\
Please update the AWS CLI and try again?\n\
  For v2: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html\n\
  For v1: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html\n"
  exit 1
fi
```

**Comment by toricls** [comment]
:)
